project/Dependencies.scala

```diff
@@ -24,9 +24,12 @@ object Dependencies {
   // Also be sure to update ScalaTestVersion in docs/build.sbt.
   val ScalaTestVersion = "3.0.4"
   val JacksonVersion = "2.8.11"
+  val JacksonCoreVersion = JacksonVersion
+  val JacksonDatatypeVersion = JacksonVersion
+  val JacksonDatabindVersion = "2.8.11.1"
```

Comment:
This particularization of Jackson versions should make it easier to maintain this list.